src/components/CounterInput/helpers/getValueOfNan.js

```diff
@@ -0,0 +1,8 @@
+const getValueOfNan = value => {
+    if (isNaN(value)) {
+        return 0;
+    }
+    return value;
+};
+
+export default getValueOfNan;
```

Comment:
I think `getValueOfNan` is confusing since you are not getting a NaN value, I think we could name it simply `getValue`